File: js-command/userinfo.js
const Discord = require("discord.js")
const bot = new Discord.Client();
const moment = require("moment")

exports.run = (bot, message) => {
    message.delete(10);
    let user;
	// If the user mentions someone, display their stats. If they just run userinfo without mentions, it will show their own stats.
    if (message.mentions.users.first()) {
      user = message.mentions.users.first();
    } else {
        user = message.author;
    }
	// Define the member of a guild.
    const member = message.guild.member(user);
	
	//Discord rich embed
    var embed = new Discord.RichEmbed()
		.setColor('RANDOM')
		.setThumbnail(user.avatarURL)
		.setTitle(`${user.username}#${user.discriminator}`)
		.addField("Nickname:", `${member.nickname !== null ? `${member.nickname}` : 'None'}`, true)
		.addField("Created At:", `${moment.utc(user.createdAt).format('dddd, MMMM Do YYYY, HH:mm:ss')}`, true)
		.addField("Joined Server:", `${moment.utc(member.joinedAt).format('dddd, MMMM Do YYYY, HH:mm:ss')}`, true)
		.addField("Bot:", `${user.bot}`, true)
		.addField("Status:", `${user.presence.status}`, true)
		.addField("Roles:", member.roles.map(roles => `${roles.name}`).join(', '), true)
     message.channel.send(embed)
     .then(sentMessage => sentMessage.delete(20000))
}

exports.help = {
    name: 'ui'
}
    

File: commands-m1/userinfo.py
@bot.command()
async def stat(ctx, member: discord.Member):
    embed = discord.Embed(title=f"{member} Info", colour=discord.Color.from_rgb(255, 215, 0))

    embed.add_field(name=f"Name - {member}")
    embed.add_field(name=f"In Guild Name - {member.name}")
    embed.add_field(name="Joined At - ", value=member.joined_at.strftime("%a, %#d %B %Y, %I:%N -> %p UTC"))
    embed.add_field(name="Created At - ", value=member.created_at.strftime("%a, %#d %B %Y, %I:%N -> %p UTC"))
    embed.add_field(name="Bot?", value=member.bot)
    embed.set_thumbnail(member.avatar_url)
    embed.set_footer(text=f"Requested By > {ctx.author.name}", icon_url=ctx.author.avatar_url)

    await ctx.send(embed=embed)